### PR TITLE
(init)set up basic env (babel, webpack, react)

### PR DESCRIPTION
**Pull request by arjun0419**
Ticket: https://trello.com/c/gL43nzHv/10-basic-initialization-react-babel-webpack

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+node_modules
+.DS_Store
+package-lock.json
```
Comment (lee-philip-n) [inline .gitignore]: package-lock.json should not be gitignored (according to Andy)

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+const path = require('path');
+
+module.exports = {
+  entry: './client/src/index.js',
```
Comment (lee-philip-n) [inline webpack.config.js]: may want to have this part as an absolute path.

**Comment by lee-philip-n** [comment]
There are two comments i made on the file changed. Reopen PR when fixed.

**Comment by arjun0419** [comment]
made those changes as per your comments. Good catch!

**Comment by arjun0419** [comment]
test comment

**Pull request merged**